tests/test_quantization/lifecycle/test_backend.py

```
@@ -0,0 +1,114 @@
+    zp = torch.tensor(0.0)
+    q_min, q_max = torch.tensor(-128.0), torch.tensor(127.0)
+    args = _args()
+
+    q = _quantize(x, scale, zp, q_min, q_max, args)
+    dq = _dequantize(q, scale, zp)
+    # dequantized value equals fused qdq
+    assert torch.allclose(dq, _quantize_dequantize(x, scale, zp, q_min, q_max, args))
+
+
+def test_set_unknown_backend_raises():
+    with pytest.raises(Exception):
```

Ruff (0.15.20)

[warning] Do not assert blind exception: `Exception`

(B017)